python/ciqueue/pytest.py

```diff
@@ -61,8 +62,22 @@ def __init__(self, config, queue):
         self.redis = queue.redis
         self.errors_key = queue.key('error-reports')
         self.terminalreporter = config.pluginmanager.get_plugin('terminalreporter')
+        if hasattr(self.terminalreporter, '_get_progress_information_message'):
+            self.__replace_progress_message()
         self.terminalwriter = config.get_terminal_writer()
 
+    def __replace_progress_message(self):
```

Comment:
This is "gravy". In pytest 3.3 they added progress reporting to the terminal:
![image](https://user-images.githubusercontent.com/1163495/34324178-fefd24f4-e836-11e7-9fcb-34b067fc1ef8.png)
This change ensures that those numbers continue to represent the progress of the entire test suite, as opposed to the % of tests run on this agent relative to the total.

Reply:
Is this output shown on the individual agents? Isn't it a little confusing to show overall progress on individual agents?

Reply:
This output is shown on the individual agents. 
Currently, the summary agent doesn't show progress. It waits until all the tests have completed before reporting on anything, and it would be a large change to alter that behaviour.

Reply:
Can you user test this? If there's confusion over what this number means we should omit it entirely. I'm worried that it will be confused as per-agent percentage and no one would guess that it represents overall build progress.

Reply:
I'm going to disable progress to unblock the crucible team and loop back in another PR (or not).